package.json

```diff
@@ -1,20 +1,37 @@
 {
-  "private": true,
-  "scripts": {
-    "prepare": "yarn tslint && lerna run prepare",
-    "rebuild:browser": "theia rebuild:browser",
-    "rebuild:electron": "theia rebuild:electron",
-    "tslint-fix": "tslint --fix --project github-extension",
-    "tslint": "tslint --project github-extension"
-  },
+  "name": "che-theia-github-plugin",
+  "publisher": "theia",
+  "keywords": [
+    "theia-plugin"
+  ],
+  "version": "0.0.1",
   "license": "EPL-2.0",
+  "contributors": [
+    {
+      "name": "Igor Vinokur",
+      "email": "ivinokur@redhat.com"
+    }
+  ],
+  "files": [
+    "src"
+  ],
   "devDependencies": {
-    "lerna": "2.4.0",
-    "tslint": "5.10.0"
+    "@octokit/rest": "15.15.1",
+    "@theia/plugin": "0.3.16",
+    "@types/node": "^10.12.0",
+    "axios": "0.18.0",
```

Comment:
I've a question : is it dev Dependencies for axios ?
Did you try to use the exported .theia file of the plug-in ? because I don't think it's working without direct dependencies

Reply:
moved to Dependencies